### ISSUE TITLE
build_typescript.sh does not run on OS X

### ISSUE THREAD

**Issue by viridia** [dimforge/rapier.js#21]
```
sed: 1: "pkg//rapier.d.ts": extra characters at the end of p command
```

The 'sed' command on OSX is the bsd version, not the gnu version. See this SO topic: https://stackoverflow.com/questions/16745988/sed-command-with-i-option-in-place-editing-works-fine-on-ubuntu-but-not-mac

**Comment by viridia** [comment]
So, I was unable to re-write the sed command to work on OSX.

However, I worked around the problem by writing a small node.js script to do the same thing. I figure if you are going to be running typescript then you have to have node installed.

**Comment by sebcrozet** [comment]
This is fixed by #26. Please reopen if you can still reproduce the problem.
